bolt/common/pcgr.py

```diff
@@ -3,14 +3,16 @@
 import re
 import shutil
 import tempfile
-
+import logging
 
 import cyvcf2
 
 
 from .. import util
 from ..common import constants
 
+# Use the existing logger configuration
+logger = logging.getLogger(__name__)
```

Comment:
Double space either side of this line; comment not needed since we'll do this as standard procedure with the upcoming logger changes